### PR TITLE
Split textand climdex indices

### PR DESCRIPTION
**Pull request by AlexSananka**
@dannyparsons This is ready to go in.

### CODE REVIEW

File: instat/sdgClimdexIndices.vb [hunk 1]
```diff
@@ -184,6 +184,9 @@ Public Class sdgClimdexIndices
         dctInputFreqPairs.Add("monthly", Chr(34) & "monthly" & Chr(34))
         ucrInputFreq.SetItems(dctInputFreqPairs)
         ucrInputFreq.cboInput.SelectedItem = "annual"
+        ucrInputFreq.SetRDefault(Chr(34) & "annual" & Chr(34))
+        ucrInputFreq.SetDropDownStyleAsNonEditable()
+        ucrInputFreq.bAllowNonConditionValues = True
```
Comment (dannyparsons) [inline instat/sdgClimdexIndices.vb]: If it's non editable we don't need to allow non condition values. Does it bug without this?

File: instat/sdgClimdexIndices.vb [hunk 2]
```diff
@@ -223,6 +226,8 @@ Public Class sdgClimdexIndices
         dctNumericPairs.Add("GSL_sum", Chr(34) & "GSL_sum" & Chr(34))
         ucrInputGSLMode.SetItems(dctNumericPairs)
         ucrInputGSLMode.cboInput.SelectedItem = "GSL"
```
Comment (dannyparsons) [inline instat/sdgClimdexIndices.vb]: Just noticed this line shouldn't be here. We shouldn't be setting the value. Can you remove this?

File: instat/sdgClimdexIndices.vb (hunk 1 above)
Comment (AlexSananka) [inline instat/sdgClimdexIndices.vb]: yes it bugs without setting the defaults

File: instat/sdgClimdexIndices.vb (hunk 2 above)
Comment (AlexSananka) [inline instat/sdgClimdexIndices.vb]: yes sure

**Comment by AlexSananka** [comment]
@dannyparsons  I have added package names and sorted out the above highlighted issues.

**Comment by dannyparsons** [comment]
Is it still giving the developer error?

**Comment by AlexSananka** [comment]
No I have now fixed them.

**Comment by dannyparsons** [comment]
I mean, is it only giving a developer error when you type something in different from the list?

**Comment by AlexSananka** [comment]
I have now do away with making it non editable and fix it by setting the "R default"

**Pull request merged**